libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -66,7 +66,11 @@ void copyMissingTags(set<CallableDeclaration const*> const& _baseFunctions, Stru
 				size_t docParaNameEndPos = content.content.find_first_of(" \t");
 				string const docParameterName = content.content.substr(0, docParaNameEndPos);
 
-				if (docParameterName != _declaration->returnParameters().at(n)->name())
+				if (
+					_declaration->returnParameterList() &&
```

Comment:
Shouldn't `_declaration->returnParameterList()` be an assertion?

Reply:
I don't think so. The latest version triggers an error already in the TagParser, but not a fatal one so this could still receive unvalidated data..